pkg/universe.dagger.io/dotnet/build.cue

```diff
@@ -0,0 +1,31 @@
+package dotnet
+
+import (
+	"dagger.io/dagger"
+)
+
+// Build a dotnet binary
+#Build: {
```

Comment:
Naming the action `#Build` when it calls `publish` could be confusing. I know you're following the existing Dagger convention, but my suggestion is to mimic the dotnet SDK command names.